tests/infrastructure/test_coverage_infrastructure.py

```
@@ -0,0 +1,3010 @@
+    def test_invalid_port(self):
+        from file_organizer.deploy.config import DeploymentConfig
+
+        with pytest.raises(ValueError, match="port"):
+            DeploymentConfig(port=99999)
+        with pytest.raises(ValueError, match="port"):
+            DeploymentConfig(port=0)
+
+    def test_data_directory_str_converts_to_path(self):
+        from file_organizer.deploy.config import DeploymentConfig
+
+        cfg = DeploymentConfig(data_directory="/tmp/data")  # type: ignore[arg-type]
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/data"

(S108)